components/admin_console/admin_definition.jsx

```diff
@@ -1,10 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+/* eslint-disable indent */
```

Comment:
Added this because eslint was stucked between `indent` and `closing tag must be aligned with opening bracket`

Reply:
Could you actually fix those indentation issues? I think it wants you to wrap them in brackets and align them like
```js
const AdminDefinition = {
    about: {
        icon: (
            <InformationOutlineIcon
                size={16}
                className={'category-icon fa'}
                color={'currentColor'}
            />
        ),
    },
};
```

Reply:
Thanks for the suggestion. Using brackets fixed the issue.